mlrun/config.py

```diff
@@ -242,6 +242,14 @@
             "channel": "master",
         },
     },
+    "storage": {
+        # What type of auto-mount to use for functions. Can be one of: none, auto, v3io_credentials, v3io_fuse, pvc.
+        # Default is auto - which is v3io_credentials when running on Iguazio and pvc otherwise (MLRun kit)
+        "auto_mount_type": "auto",
+        # Extra parameters to pass to the mount call (will be passed as kwargs). Parameters should be provided as a
+        # string of comma-separated parameters, using this format: "param1=value1,param2=value2"
+        "auto_mount_params": "",
```

Comment:
Make it a base64 json string to allow values to have `,` and `=` and also to be off non-primitive types (dict, list) 
Even though we probably don't need it right now
See `get_default_function_node_selector` for how we did it already for other config values

Reply:
It makes it more difficult to encode values, for example if all I want is to say `"pvc_name=a-name"`. I do like the idea, though, so I implemented both ways (and testing both).